### ISSUE TITLE
Which License?

### ISSUE THREAD

**Issue by ryoppippi** [gruhn/typescript-sudoku#3]
Awesome project! Great Job!
BTW, which license you apply for this project

**Comment by gruhn** [comment]
I don’t know 🤷‍♂️ MIT I guess. Does it matter for you?

**Comment by ryoppippi** [comment]
Yes, I asked a question because I don't know if I can use the code here freely. It's an important issue.

There is a possibility that the person who downloaded the code here will be asked to pay.

The license is to make it clear
